packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -586,13 +586,13 @@ async function handler(req: CustomRequest) {
     }
 
     // Posible to refactor TODO:
-    const paymentApp = await appStore[paymentAppCredential?.app?.dirName as keyof typeof appStore];
+    const paymentApp = await appStore[paymentAppCredential?.app?.dirName as keyof typeof appStore]();
     if (!(paymentApp && "lib" in paymentApp && "PaymentService" in paymentApp.lib)) {
       console.warn(`payment App service of type ${paymentApp} is not implemented`);
       return null;
     }
-
-    const PaymentService = paymentApp.lib.PaymentService;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This is possibly due to the interface not having a constructor method, but outside the scope of this experiment.